updated scRNA user guide (#475)
NCI-GDC/gdc-docs: docs/Data_Portal/Users_Guide/scRNA.md

  * Presents variance distribution across components.

**Note:**

> To display a certain plot type, check its respective box. You can examine each plot separately. When multiple plot types are open, they can be viewed simultaneously. You can use the “Basics of Plots” section controls in all three dimensionality reduction plot types.

Reviewer: Could you change these quotation marks to the plain ASCII ones: "  

The special quotation marks do not render when built into the doc site.

Author: Sure I can change those.

> To display a certain plot type, check its respective box. You can examine each plot separately. When multiple plot types are open, they can be viewed simultaneously. You can use the “Basics of Plots” section controls in all three dimensionality reduction plot types.

Reviewer: ```suggestion
To display a certain plot type, check its respective box. You can examine each plot separately. When multiple plot types are open, they can be viewed simultaneously. You can use the "Basics of Plots" section controls in all three dimensionality reduction plot types.
```

#### Color Scale Configuration

> Users can configure the color of the expression legend through the minimum and maximum selectors when clicking on the color bar. For both the min and max, the tab presents a color selection window enabling precise control of the color and hue that are selected. . A user can also select Automatic, Fixed, or Percentile to set the min and max.

Reviewer: There seems to be an extra period here.  
```suggestion
Users can configure the color of the expression legend through the minimum and maximum selectors when clicking on the color bar. For both the min and max, the tab presents a color selection window enabling precise control of the color and hue that are selected. A user can also select Automatic, Fixed, or Percentile to set the min and max.
```

### Gene Set Enrichment Analysis (GSEA)

Users can examine the pathways that are most enriched in samples through the GSEA tab after a DE analysis is run. Users can select various options to refine their analysis:
*	Number of permutations (a higher number gives higher accuracy but with a longer runtime).
*	Minimum and maximum gene set size filter to control the size of the gene sets used in the analysis.
*	Filtering non-coding genes from the analysis.

> *	Choosing a cutoff parameter controls whether FDR or ‘top gene sets’ is used as the cutoff for gene sets. Specifically, the cutoff method allows the user to set a specific number of gene sets ordered by their ascending FDR value.

Reviewer: ```suggestion
*	Choosing a cutoff parameter controls whether FDR or 'top gene sets' is used as the cutoff for gene sets. Specifically, the cutoff method allows the user to set a specific number of gene sets ordered by their ascending FDR value.
```

[![Violin plot](./images/scrna/image18-summary.png)](./images/scrna/image18-summary.png 'Click to see the full image.')

Measures of central tendency such as mean, median, variance, and others are provided, along with a table showing pairwise statistical comparisons. This comparison is performed between all clusters for the particular gene of interest, with their respective p-values (generated by the Wilcoxon Rank Sum test method). Several options to refine the violin plot visualization include:

> 1.	The ability to alter the orientation of the plot (horizontal or vertical).

Reviewer: Add a return before this line.  I think the numbering will not render as a list without a return.

Reviewer: I'm not sure if this matters, but the tab between the numbers and the steps might cause an issue.  I recommend changing them to spaces.
